app/store/middlewares/helpers.js

```diff
@@ -39,17 +39,52 @@ export function cleanUpState(payload, keepCurrent = false) {
             files: {},
             fileIdsByPostId: {},
         },
+        general: {
+            ...payload.entities?.general,
+            dataRetention: {
+                ...payload.entities?.general?.dataRetention,
```

Comment:
Previous app does not have this state.. what is the expected behavior on upgrading the app?

Reply:
Added code for migrating from old data to the changed one.